testing/integration/logs_ingestion_test.go

```diff
@@ -179,6 +178,10 @@ func testMonitoringLogsAreShipped(
 		"Looking for logs in dataset 'elastic_agent.metricbeat'")
 
 	// Stage 2: make sure all components are healthy
+	t.Logf("Getting Agent ID")
+	agentID, err := agentFixture.AgentID(ctx)
```

Comment:
do we need this call here? right below we invoke directly `agentFixture.ExecStatus(ctx)`. Can't we extract agentID from there?

Reply:
Updated it to use, the the result of `agentFixture.ExecStatus(ctx)`